src/app/components/AssignmentDetails.tsx

```diff
@@ -16,33 +16,32 @@ export interface AssignmentDetailsProps {
 
 export default class AssignmentDetails extends React.Component<AssignmentDetailsProps, any>{
     render() {
-        const { assignment } = this.props;
+        const { assignment : {startTime, endTime, workSectionId, title, assignmentCourt, pickupLocation, dropoffLocation, sherrifsRequired, gateNumber, notes } } = this.props;
         return (
             <div>
-                <h2>{assignment.assignmentType}</h2>
-                <strong>Start Time: </strong>{assignment.startTime.toString().substring(0, 24)}
+                <h2>{title}</h2>
+                <strong>Start Time: </strong>{startTime.toString().substring(0, 24)}
```

Comment:
I would suggest using moment for displaying dates / times.  Moment takes care of all the parsing and date math for you as well as providing methods for displaying nice human readable date/times.